src/components/cards/CommentCard.tsx

```diff
@@ -0,0 +1,245 @@
+import React from "react";
+import { FontCSS, TdCSS } from "../../css";
+import { palette } from "@guardian/src-foundations";
+import { Content, Tag } from "../../api";
+import { formatImage } from "../../image";
+import sanitizeHtml from "sanitize-html";
+import { Table, RowCell, TableRowCell, TableRow } from "../../layout/Table";
+import { Headline } from "../../components/Headline";
+import { Image } from "../../components/Image";
+import { headline } from "../../styles/typography";
+
+type Size = "small" | "large";
+
+const tdStyle: TdCSS = {
+    backgroundColor: palette.neutral[100],
+    borderTop: `1px solid ${palette.opinion.main}`,
+    padding: "0"
```

Comment:
`padding: "0"` unnecessary - it's added as default as we're using the TableRow component.

Reply:
padding removed